src/contracts/core/DelegationManager.sol

```diff
@@ -19,6 +21,35 @@ import "./DelegationManagerStorage.sol";
  * - enabling a staker to undelegate its assets from the operator it is delegated to (performed as part of the withdrawal process, initiated through the StrategyManager)
  */
 contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, DelegationManagerStorage, ReentrancyGuardUpgradeable {
+
+    function operatorShares(address operator, IStrategy strategy) public view returns (uint256) {
+        uint256 scalingFactor = slasher.shareScalingFactor(operator, strategy);
+        return SlashingAccountingUtils.scaleDown(nonNormalizedOperatorShares[operator][strategy], scalingFactor);
+    }
+
+    // includes the effect of all unexecuted but pending slashings
+    function operatorSharesIncludingPendingSlashings(address operator, IStrategy strategy) external view returns (uint256) {
+        uint256 scalingFactor = slasher.pendingShareScalingFactor(operator, strategy);
+        return SlashingAccountingUtils.scaleDown(nonNormalizedOperatorShares[operator][strategy], scalingFactor);
+    }
+
+    // TODO: decide if this function needs to go in the interface at all
+    function pendingWithdrawalData(bytes32 withdrawalRoot) public view returns (PendingWithdrawalData memory) {
+        return _pendingWithdrawalData[withdrawalRoot];        
+    }
+
+    // @notice Returns 'true' if the `withdrawalRoot` corresponds to a queued-but-not-completed withdrawal, and 'false' otherwise
+    function pendingWithdrawals(bytes32 withdrawalRoot) public view returns (bool) {
```

Comment:
`isWithdrawalPending`

Reply:
this is backwards compatible 🤷 
I agree the name you suggest would be better though

Reply:
ah rip nice